### PR TITLE
#104 Jquery convert to pure javascript

### PR DESCRIPTION
**Pull request by AgriyaDev5**
## PR description

 Jquery convert to pure javascript

### CODE REVIEW

File: src/editor/extensions/ext-connector/ext-connector.js [hunk 1]
```diff
@@ -393,7 +393,7 @@ export default {
 
           const parents = svgCanvas.getParents(mouseTarget.parentNode);
 
-          if ($.inArray(svgcontent, parents) !== -1) {
+          if (parents.indexOf(svgcontent) !== -1) {
```
Comment (jfhenon) [inline src/editor/extensions/ext-connector/ext-connector.js:396]: Does indexOf works with DOM elements?

**Review by jfhenon — COMMENTED** [review]
see question

File: src/editor/extensions/ext-imagelib/index.js [hunk 2]
```diff
@@ -41,10 +40,22 @@ Array.prototype.forEach.call(atags, function (aEle) {
       });
       img.src = href;
     } else {
+      const xhr = new XMLHttpRequest();
```
Comment (jfhenon) [inline src/editor/extensions/ext-imagelib/index.js:43]: why not using fetch here?

File: src/editor/extensions/ext-connector/ext-connector.js (hunk 1 above)
Comment (AgriyaDev5) [reply src/editor/extensions/ext-connector/ext-connector.js]: indexOf change to includes. it's working

File: src/editor/extensions/ext-imagelib/index.js (hunk 2 above)
Comment (AgriyaDev5) [reply src/editor/extensions/ext-imagelib/index.js]: modified

**Review by jfhenon — APPROVED** [review]

**Pull request merged**